### PR TITLE
[RFR] Basic pageview Tracking implementation

### PR DESCRIPTION
**Pull request by Tom-Bonnike**

### CODE REVIEW

File: src/routes.js [hunk 1]
```diff
@@ -1,7 +1,10 @@
+/* eslint no-undef: "off" */
```
Comment (Tom-Bonnike) [inline src/routes.js]: Required as eslint will trigger an error for the undefined property “CustomEvent” in this file. CustomEvent is only defined on the clientside.

File: src/routes.js [hunk 2]
```diff
@@ -12,15 +15,35 @@ import EditPetition from 'containers/EditPetition';
 import PublishedPetition from 'containers/PublishedPetition';
 import RespondToPetition from 'containers/RespondToPetition';
 
+const pageviewEvent = __CLIENT__ &&
+  typeof CustomEvent !== 'undefined' && // Required as it’s not defined during tests.
+  new CustomEvent('IRIS_pageview');
```
Comment (Tom-Bonnike) [inline src/routes.js]: Is it normal that __CLIENT__ is set to true during some tests?

**Comment by eschaefer** [comment]
Ok something is up with Travis since I added `SITE_NAME` to the heroku app's env config, and re-builds don't help.

**Comment by eschaefer** [comment]
Moved to https://github.com/iris-dni/iris-frontend/pull/150

**Pull request closed without merging**